### PR TITLE
DBZ-1241 - allow null values again for commitTime

### PR DESCRIPTION
**Pull request by jorkzijlstra**
@gunnarmorling Could you verify if this change doesn't break the [DBZ-1082](https://github.com/debezium/debezium/commit/8e21139ca3ee140a07b99cbfb87b6a92b1253308#diff-f28bb7eecf710b4c4de5235a03902776R181)?

### CODE REVIEW

**Comment by jpechane** [comment]
@jorkzijlstra Thanks, LGTM!
Could you please just extend `io.debezium.connector.postgresql.SourceInfoTest` with a test method that will call `io.debezium.connector.postgresql.SourceInfo.update(Long, Long, Instant, Long, TableId, Long)` with all parameters set to `null` so it is verified that it works?

Also thinking about it - in `io.debezium.connector.postgresql.connection.wal2json.StreamingWal2JsonMessageDecoder.outOfOrderChunk(byte[])` the `commitTime` should be initilized to current time insted of null.

**Comment by gunnarmorling** [comment]
> commitTime should be initilized to current time insted of null.

Interesting, wouldn't this be more confusing? If we don't have the metadata, why pretend we do? It seems the code must generally be prepared to work without having a value for the commit time?

**Comment by jorkzijlstra** [comment]
I have added the test setting all null values.

As for the default value setting it to current time. I agree with @gunnarmorling that it would be more confusing.

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/pgproto/PgProtoReplicationMessage.java [hunk 1]
```diff
@@ -70,8 +70,13 @@ public Operation getOperation() {
 
     @Override
     public Instant getCommitTime() {
-        // value is microseconds
-        return Instant.ofEpochSecond(0, rawMessage.getCommitTime() * 1_000);
+        Long commitTime = rawMessage.getCommitTime();
+        if(commitTime != null) {
+            // value is microseconds
+            return Instant.ofEpochSecond(0, commitTime * 1_000);
+        } else {
```
Comment (gunnarmorling) [inline debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/pgproto/PgProtoReplicationMessage.java]: `rawMessage.getCommitTime();` above returns a primitive `long`,  i.e. the `Long commitTime` actually will never be `null` and this branch is dead code. It seems the original version was just fine?

File: debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/SourceInfoTest.java [hunk 2]
```diff
@@ -34,4 +35,10 @@ public void versionIsPresent() {
     public void connectorIsPresent() {
         assertThat(source.source().getString(SourceInfo.DEBEZIUM_CONNECTOR_KEY)).isEqualTo(Module.name());
     }
+
+    @Test(expected = Test.None.class /* no exception expected */)
```
Comment (gunnarmorling) [inline debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/SourceInfoTest.java]: Why not just omit the `expected` then if no exception is expected?

Comment (jorkzijlstra) [reply debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/SourceInfoTest.java]: I didn't know that existed

Comment (jorkzijlstra) [reply debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/SourceInfoTest.java]: I'm used to scalatest where you write out that you except no exception

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/pgproto/PgProtoReplicationMessage.java (hunk 1 above)
Comment (jorkzijlstra) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/pgproto/PgProtoReplicationMessage.java]: You know this code better than me, Ill revert it

**Comment by jorkzijlstra** [comment]
@gunnarmorling I was working on my company laptop with github enterprise user. If fixed it now and my account has been properly connected. So my name is indeed Jork Zijlstra.

File: debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/SourceInfoTest.java (hunk 2 above)
Comment (gunnarmorling) [reply debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/SourceInfoTest.java]: Ah, I see. Never used Scalatest before. Always learning something new here :)

**Comment by gunnarmorling** [comment]
Thanks, @jorkzijlstra! LGTM now; will wait for CI to pass and then merge.

**Comment by jorkzijlstra** [comment]
@gunnarmorling Thnx, although its a small change, thnx for all the help in getting it right :)

**Comment by gunnarmorling** [comment]
Applied. Thanks a lot for your contribution, @jorkzijlstra!

**Pull request merged**